src/__tests__/components/TransactionListTop.test.tsx

```diff
@@ -52,7 +52,8 @@ describe('TransactionListTop', () => {
     core: {
       account: {
         currencyWallets: { '123': fakeWallet },
-        currencyConfig: { bitcoin: fakeCurrencyConfig }
+        currencyConfig: { bitcoin: fakeCurrencyConfig },
+        watch() {}
```

Comment:
Why is this `watch()` added for this commit? Seems unrelated; not sure what's going on here.

Reply:
context

Reply:
fixed